src/test/java/org/broadinstitute/consent/http/service/VoteServiceTest.java

```diff
@@ -787,78 +733,7 @@ public void testNotifyCustodiansOfApprovedDatasets() {
     researcher.setDisplayName("Researcher");
     researcher.setUserId(1);
 
-    User depositor = new User();
-    depositor.setEmail("depositor@test.com");
-    depositor.setDisplayName("depositor");
-    depositor.setUserId(2);
-
-    User custodian = new User();
-    custodian.setEmail("custodian@test.com");
-    custodian.setDisplayName("custodian");
-    custodian.setUserId(3);
-
     when(userDAO.findUserById(any())).thenReturn(submitter);
-    when(userDAO.findUsersByEmailList(any())).thenReturn(List.of(depositor, custodian));
-
-    initService();
-    try {
-      service.notifyCustodiansOfApprovedDatasets(List.of(d1, d2), researcher, "Dar Code");
-      verify(emailService, times(1)).sendDataCustodianApprovalMessage(
-          any(),
-          any(),
-          any(),
-          any(),
-          any()
-      );
-    } catch (Exception e) {
-      fail(e.getMessage());
-    }
-  }
-
-  @Test
-  public void testNotifyCustodiansOfApprovedDatasetsWithCustodianProps() {
```

Comment:
Note that this test duplicates `testNotifyStudyCustodiansAndSubmittersOfApprovedDatasets` but incorrectly. It uses `dataCustodianEmail` as a dataset property instead of what it should be, a study property. `testNotifyStudyCustodiansAndSubmittersOfApprovedDatasets` exercises the correct behavoir.